src/compiler/transformers/ts.ts

```diff
@@ -1355,7 +1362,9 @@ namespace ts {
                     )
                 );
 
-                return createAssignment(getDeclarationName(node), expression);
+                const result = createAssignment(getDeclarationName(node), expression, getUndecoratedRange(node));
+                setNodeEmitFlags(result, NodeEmitFlags.NoComments);

```

Comment:
why we do ` NodeEmitFlags.NoComments` for decorated class? why not do ` context.setNodeCustomCommentRange(method, node);`?